packages/frontend/components/item/image-slider/component.tsx

```diff
@@ -8,38 +8,58 @@ interface ItemImageSliderProps {
 
 export const ItemImageSlider = ({ imageLinks }: ItemImageSliderProps) => {
   const [focusedImage, setFocusedImage] = useState(0);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const imagesWrapper = useRef<HTMLDivElement>(null);
 
   const handleClick = (imageLink) => {
     setFocusedImage(imageLink);
   };
 
+  useEffect(() => {
+    function handleWheelScroll(e) {
+      e.preventDefault();
+      imagesWrapper.current.scrollLeft += e.deltaY;
+      imagesWrapper.current.scrollTop += e.deltaY;
+    }
+    imagesWrapper.current.addEventListener('wheel', handleWheelScroll);
+  }, []);
+
   return (
     <div css={styles.sliderWrapper}>
-      <div css={styles.imagesWrapper}>
+      <div css={styles.imagesWrapper} ref={imagesWrapper}>
         {imageLinks.map((link, index) => (
           <div
             key={link + index}
-            css={[styles.image, index === focusedImage && styles.pickedImage]}
+            css={[
+              styles.imageWrapper,
+              index === focusedImage && styles.pickedImage,
+            ]}
+            onClick={() => handleClick(index)}
+            aria-hidden="true"
           >
-            <Image
-              key={index}
-              onClick={() => handleClick(index)}
-              src={link}
-              alt="item image"
-              layout="fill"
-              objectFit="cover"
-            />
+            <img key={index} src={link} alt="item" css={styles.image} />
           </div>
         ))}
       </div>
-      <div css={styles.focusedImage}>
-        <Image
-          src={imageLinks[focusedImage]}
-          alt="item image"
-          layout="fill"
-          objectFit="cover"
-        />
+      <div
+        css={styles.focusedImageWrapper}
+        onClick={() => setIsModalOpen(true)}
```

Comment:
It's bad practice to have clickable div. But you are working in that way, no need to provide role, tabIndex and onKeyDown handler